tests/scripts/quiet/cmake

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env bash
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+#
+# This swallows the output of the wrapped tool, unless there is an error.
+# This helps reduce excess logging in the CI.
+
+# If you are debugging a build / CI issue, you can get complete unsilenced logs
+# by un-commenting the following line (or setting VERBOSE_LOGS in your environment):
+# VERBOSE_LOGS=1
+
+# don't silence invocations containing these arguments
+NO_SILENCE=" --version "
+
+TOOL=$(basename "$0")
+
+# Locate original tool
+ORIGINAL_TOOL=$(type -ap ${TOOL} | grep -v "$0" | head -n1 )
+
+quote_args() {
+    # similar to printf '%q' "$@"
+    # but produce more human-readable results for common/simple cases like "a b"
+    local args=("$@")
+    s=""
+    for a in "${args[@]}"; do
+        simple_pattern='^[[:alnum:] _=+-]*$'
+        if [[ $a =~ ' ' && $a =~ $simple_pattern ]]; then
+            # a has spaces, but no other special characters that need escaping
+            # (quoting after removing spaces yields no backslashes)
+            # simplify quoted form to "$a" - e.g. yield "a b c" instead of a\ b\ c
+            q="\"$a\""
+        else
+            # get bash to do the quoting
+            q=$(printf '%q' "$a")
+        fi
+        s="$s $q"
+    done
+    echo $s
+}
+
+if [[ ! " $@ " =~ " --version " ]]; then
```

Comment:
Minor: `" $@ "` is correct but weird: the difference between `$*` and `$@` is that `"<PREFIX>$@<SUFFIX>"` is multiple words when used in list context, but here the string is in word context so the result is the same as `" $* "`: concatenation of the arguments with a space in between. Since the concatenation is intended, it's less surprising to write `" $* "`.